src/pages/Marketplace/Cards/useCards.ts

```diff
@@ -67,27 +67,31 @@ export default function useCards() {
   async function loadNextPage() {
     //button is hidden when there's no more
     if (
-      data?.ItemCutoff &&
+      data?.Page &&
       originalArgs /** cards won't even show if no initial query is made */
     ) {
       const { data: newEndowRes } = await loadMore({
         ...originalArgs,
-        start: data.ItemCutoff + 1,
+        page: data.Page + 1,
       });
 
       if (newEndowRes) {
         //pessimistic update to original cache data
         dispatch(
           updateAWSQueryData("endowmentCards", originalArgs, (prevResult) => {
             prevResult.Items.push(...newEndowRes.Items);
-            prevResult.ItemCutoff = newEndowRes.ItemCutoff;
+            prevResult.Page = newEndowRes.Page;
           })
         );
       }
     }
   }
 
-  const hasMore = !!data?.ItemCutoff;
+  // initial assumption is that there's no more to load until we get first res from query
+  let hasMore = false;
+  if (data?.Page !== undefined && data?.NumOfPages !== undefined) {
+    hasMore = data.Page < data.NumOfPages;
```

Comment:
```suggestion
    const hasMore = (data?.Page || 0) < (data?.NumOfPages || 0);
```
nit: more declarative expression